env0/resource_azure_credentials.go

```diff
@@ -0,0 +1,101 @@
+package env0
+
+import (
+	"context"
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAzureCredentials() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAzureCredentialsCreate,
+		ReadContext:   resourceAzureCredentialsRead,
+		DeleteContext: resourceAzureCredentialsDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Description: "name for the credentials",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"client_id": {
+				Type:        schema.TypeString,
+				Description: "the azure client id",
+				Required:    true,
+				Sensitive:   true,
+				ForceNew:    true,
+			},
+			"client_secret": {
+				Type:        schema.TypeString,
+				Description: "the azure client secret",
+				Required:    true,
+				Sensitive:   true,
```

Comment:
Technically this is the only field that's actually sensitive, but I don't think that matters much
I'm fine with keeping all of those `Sensitive: true`